CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 * 7.10.2 as default version.
 
+
+| PR | Author | Title |
+| --- | --- | --- |
+| [#123](https://github.com/elastic/ansible-beats/pull/123) | [@jmlrt](https://github.com/jmlrt) | Cleanup init_script variable  |
```

Comment:
Do you know https://github.com/elastic/ansible-beats/commit/99efe3ba672ab41559d9c6187a358ae1a96ff826 is missing here?